action.yml

```diff
@@ -13,6 +13,11 @@ inputs:
     description: 'YARD runner will be executed with this ruby version'
     required: true
     default: '3.2'
+outputs:
+  page_url:
+    description: "github page url"
+    value: ${{ steps.deployment.outputs.page_url }}
```

Comment:
Which behaviors will use this one? I know actions/deploy-pages@v1 outputs as this. However personally I don't use this...

Reply:
[the deployment environment](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)